docs/installation/configuration/env_config.md

```diff
@@ -127,6 +127,13 @@ The following parameters control the CORS policy.
   included. The value of this variable is added to these already included headers.
   Defaults to an empty list.
 
+### Initial configuration
+
+Open Notificaties supports `setup_configuration` management command, which allows configuration via
+environment variables. 
+All these environment variables are described at CLI configuration.
```

Comment:
Can we add a link here to the `opennotifs_config_cli`? Might be an issue because it's a markdown file and not a reStructuredText file (not sure why that is the case :thinking:)

Reply:
Yes, I tried to add link here, but I don't think .md supports it. And I really didn't want to rewrite this whole file in .rst

Reply:
I'll create a separate issue for it, this same file in Open Zaak currently has an `.rst` style link which doesn't work

Reply:
Thank you!